cli/lib/commands/help/--version.txt

```diff
@@ -0,0 +1,4 @@
+Usage: ds --version
```

Comment:
Could we throw in an alias to `-v` somehow? I keep on typing `ds -v` and then making a sad face.